packages/contracts-bedrock/test/L2/L2StandardBridge.t.sol

```diff
@@ -130,13 +131,42 @@ contract L2StandardBridge_Test is Bridge_Initializer {
         assertEq(address(l2ToL1MessagePasser).balance, 100);
     }
 
+    /// @dev Tests that the receive function reverts with custom gas token.
+    function test_receive_customGasToken_reverts() external {
+        vm.prank(alice, alice);
```

Comment:
This could be a good fuzz test with value

Reply:
I thought we'd just revert for all values with custom gas token? if so, I don't think it'd make sense to fuzz?